README.md

```diff
@@ -1 +1,28 @@
-# hw01
```

Comment:
Topic | Excellent | Satisfactory | Needs Work
-- | -- | -- | --
Coding style | X |   |  
Coding strategy | X |   |  
Presentation | X |   |  
Achievement, creativity | X |   |  
Ease of access | X